README.md

```diff
@@ -30,15 +30,19 @@ To contribute:
 | [JPMorgan](https://jpmchase.taleo.net/careersection/10140/jobdetail.ftl?job=190064684)| | Summer|October 27, 2019|
 | [Jump Trading](https://www.jumptrading.com/jobs.html)  | Downtown | Summer | |
 | [Lazada](http://www.lazada.com/work-at-lazada) | Tanjong Pagar | Summer | |
+| [Microsoft](https://careers.microsoft.com/us/en/job/697177/2020-Summer-Internship-Opportunities-Singapore) | #13-01 Frasers Tower | Summer | Deadline: 1 Dec 2019 |
```

Comment:
This sounds a lot more like an internship geared towards MBA students. They seem to be open to Bachelor's too but the description and requirements do not suggest that the applicant will be working on anything technical.

Reply:
Oops, you're right. I'll remove it.